index.html

```diff
@@ -307,10 +307,10 @@ <h3>History</h3>
                     <div class="feature-description">
                         <p><strong>Go back in time</strong></p>
                         <ul>
-                            <li>Watch your favourites again</li>
-                            <li>Search suggestions based on your last searches</li>
-                            <li>Easily open what you watch most</li>
-                            <li class="text-success">Control what we keep for you</li>
+                            <li>Watch favourites again</li>
+                            <li>Search-suggestions based on your last searches</li>
```

Comment:
I think this was correct as is.

Reply:
True, but I added it to avoid ~ Search for/in suggestions based on your last searches"